libs/ui/src/molecules/Footer/Footer.tsx

```diff
@@ -0,0 +1,112 @@
+
+import Image from "next/image";
+import { FaTwitter, FaDiscord, FaTelegram } from "react-icons/fa";
+
+const logo = require('../../assets/images/logo.png');
+
+const Footer = () => {
+  return (
+    <footer style={{ background: `rgb(242,55,62) linear-gradient(179deg, rgba(242,55,62,1) 0%, rgba(33,54,174,1) 99%, rgba(34,54,175,1) 100%)` }}>
+      <div className="mx-auto max-w-7xl">
+        {/* First layer */}
+        <div className="relative flex h-44 pt-12">
+            <div className="items-center px-2 lg:px-0">
+              <Image
+                className="hidden h-8 w-auto lg:block"
+                src={logo}
+                alt="Your Company"
+              />
+              <div className="pt-4">
+                <div className="rounded-md text-sm font-medium text-white">
+                    Discover, collect, and sell digital items on the
+                </div>
+                <div className="rounded-md text-sm font-medium text-white">
+                  largest NFT marketplace on Optimism
+                </div>
+              </div>
+            </div>
+            <div className="flex flex-1 justify-center lg:justify-end">
+              <div className="lg:mr-20">
```

Comment:
i know in the design, the right side doesn't exactly line up with the rest of the footer content below it, but check with Daru because i'm not sure if that was intentional or not. we might want to take out the mr-20 class